spring-pulsar/src/main/java/org/springframework/pulsar/listener/DefaultPulsarMessageListenerContainer.java

```diff
@@ -602,6 +622,18 @@ private void handleAck(Message<T> message) {
 			AbstractAcknowledgement.handleAckByMessageId(this.consumer, message.getMessageId());
 		}
 
+		public void pause() {
```

Comment:
As we have `pause/resume` in several places, is there already a Spring abstraction for it? If not, should we create a `Pauseable` interface and use accordingly?

Reply:
Let's do that outside the scope of this PR.